### PR TITLE
add jekyll guide

### LINKED ISSUE #323: Guide for Jekyll
Hey, i am planning to write a guide for jekyll. It's a static site generator based on ruby.

### PR DESCRIPTION
**Pull request by pclnwk**
Add a guide to install jekyll on uberspace

Closes #323

### CODE REVIEW

File: source/guide_jekyll.rst [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+ [isabell@stardust ~]$
+
+Installation
+============
+Jekyll is a ruby gem that creates dummy files and directories for us that contain your website. We can modify these files to make your website look like we want it to.
+
+Just let Jekyll create a new folder containing your website by typing:
+::
+
+ [isabell@stardust ~]$ jekyll new website
```
Comment (luto) [inline source/guide_jekyll.rst]: Just of out curiosity: this already seems to install the needed gems. They are even present in `~/website/vendor/bundle/gems/`, but `bundle exec` can't find the `jekyll` command. Do you happen to know why?

**Comment by luto** [comment]
Thank you and sorry about the delay!

File: source/guide_jekyll.rst (hunk 1 above)
Comment (pclnwk) [reply source/guide_jekyll.rst]: Hey, I am currently on vacation and have no possibility to check that out on my uberspace. They blocked ssh ports on guest wifi. I will have a look at it, when I am back.

Comment (pclnwk) [reply source/guide_jekyll.rst]: I tried to reproduce your question and figured out, that on a local machine the command `jekyll new website` is enough to execute `bundle exec jekyll serve` and everything works fine.
But on uberspace I have to type `bundle install --path vendor/bundle` to install missing gems after creating the new website. 
But I dont know why the gems seem to be missing. 

I try to figure out why and if I find a reason, I will comment again.

**Pull request merged**